src/app/components/client/exposure_card/ScanResultCard.tsx

```diff
@@ -141,38 +164,17 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
           },
         );
       case "new":
-        // Data broker cards manually resolved do not change their status to "removed";
-        // instead, we track them using the "manually_resolved" property.
-        if (scanResult.manually_resolved) {
+        /* c8 ignore start */
+        if (props.isOnManualRemovePage) {
           return l10n.getFragment(
-            "exposure-card-description-info-for-sale-fixed-manually-fixed",
+            "exposure-card-description-info-for-sale-action-needed-manual-fix-page",
             {
               elems: {
                 data_broker_profile: dataBrokerProfileLink,
               },
             },
           );
         }
-        /* c8 ignore start */
-        if (props.isOnManualRemovePage) {
-          return scanResult.manually_resolved
```

Comment:
This variant is still being handled [here](https://github.com/mozilla/blurts-server/pull/4700/files#diff-8e5a241da4de579ab3a4b5747a3f7dd6ff339d114b78948c35d93a55ebfcaf18R168). The diff is a bit confusing, because I’ve moved the `manually_resolved` check up: https://github.com/mozilla/blurts-server/pull/4700/files#diff-8e5a241da4de579ab3a4b5747a3f7dd6ff339d114b78948c35d93a55ebfcaf18R126-R135.